src/AutoRest.CSharp/Common/Output/Models/Shared/KnownParameters.cs

```diff
@@ -58,5 +60,9 @@ internal static partial class KnownParameters
             DefaultValue: null,
             Validation: ValidationType.None,
             Initializer: null);
+        public static bool TypeAndNameMatch(Parameter parameter1, Parameter parameter2)
```

Comment:
is this needed? can we use the existing one？
https://github.com/Azure/autorest.csharp/blob/268545602af22109c028c7c7316fc15a056c1d12/src/AutoRest.CSharp/Common/Output/Models/Shared/Parameter.cs#L25